mesonbuild/backend/ninjabackend.py

```diff
@@ -57,6 +57,20 @@
     execute_wrapper = []
     rmfile_prefix = ['rm', '-f', '{}', '&&']
 
+def get_rsp_threshold():
+    '''Return a conservative estimate of the commandline size in bytes
+    above which a response file should be used.  May be overridden for
+    debugging by setting environment variable MESON_RSP_THRESHOLD.'''
+
+    if mesonlib.is_windows():
+        limit = 8192
+    else:
+        limit = 131072
+    return int(os.environ.get('MESON_RSP_THRESHOLD', limit/2))
```

Comment:
I think the `/2` here is worthy of comment.

Reply:
Yes.  Fixed.